pkg/convert/go_messages.go

```diff
@@ -0,0 +1,51 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"golang.org/x/text/message/pipeline"
+)
+
+func FromGoMessages(m model.Messages) ([]byte, error) {
+	pipelineMsg := MessagesToPipeline(m)
+
+	msg, err := json.Marshal(pipelineMsg)
+	if err != nil {
+		return []byte{}, fmt.Errorf("error while marshaling: %w", err)
+	}
+
+	return msg, nil
+}
+
+func ToGoMessages(b []byte) (pipeline.Messages, error) {
+	var msg pipeline.Messages
+
+	err := json.Unmarshal(b, &msg)
+	if err != nil {
+		return pipeline.Messages{}, fmt.Errorf("error while unmarshaling: %w", err)
+	}
+
+	return msg, nil
+}
+
+func MessagesToPipeline(m model.Messages) pipeline.Messages {
+	pipelineMsg := pipeline.Messages{}
```

Comment:
Declare and initialize all values in one go

```suggestion
	pipelineMsg := pipeline.Messages{Language: m.Language}
```